_data/versions.yml

```diff
@@ -1346,6 +1346,67 @@ v2.1:
       version: 1.4.1
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.1
 
+v3.0:
+- title: v3.0.0-beta-rc1
+  note: |
+    20 October 2017
+
+    Calico v3.0.0-beta adds fixes and enhancements to the Kubernetes integration. Enhancements include:
+    - Support for etcdv3 datastore
+    - A new API tuned for etcdv3 - see [libcalico-go release notes](https://github.com/projectcalico/libcalico-go/releases) for more details.
+
```

Comment:
^^ + `- Named ports`

Reply:
We could also make this a link or provide a link in some other way to the docs that describe this feature, which appear to be here: https://docs.projectcalico.org/master/reference/calicoctl/resources/hostendpoint#endpointport and here https://docs.projectcalico.org/master/reference/calicoctl/resources/workloadendpoint#endpointport

ofc, when we release, they will be here https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/hostendpoint#endpointport and here https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/workloadendpoint#endpointport (<--- better links)